src/patternfly/components/Chip/examples/chip-complex-example.hbs

```diff
@@ -0,0 +1,128 @@
+{{#> chip-group chip-group--attribute='role="group of chips"'}}
```

Comment:
The `role` attribute can only take a single value. All acceptable values are listed [here](https://www.w3.org/TR/wai-aria-1.1/#role_definitions).

I really want to suggest that the appropriate role/element for this is a list. And maybe all chips should be `<li>` elements, whether they are styled as a group or not. 

The part that makes using a list seem weird is the "4 more" button, since the main benefit of using a list is to group elements together and provide a count, so if up front you get a count of 3, but the true count is 6, that might be confusing. There are [aria attributes](https://www.w3.org/TR/wai-aria-1.1/#aria-posinset) that allow us to provide more indication about the number of list items, but they don't seem to be supported very well (at least not in NVDA and VO).